src/main/java/enums/Command.java

```diff
@@ -0,0 +1,18 @@
+package enums;
+
+public enum Command {
+    BYE,
+    LIST,
+    MARK,
+    UNMARK,
+    DELETE,
+    TODO,
+    DEADLINE,
+    EVENT,
+    FIND;
+
+    @Override
```

Comment:
Same as above for overridden methods, since you are changing the behaviour of the method.